### PR TITLE
[2.1] Php7 list() syntax

### PR DESCRIPTION
**Pull request by Faryshta**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | yes
| Tests pass?   | yes
| Fixed issues  | none but related to #11397

### CODE REVIEW

File: framework/db/ActiveQuery.php [hunk 1]
```diff
@@ -155,7 +155,7 @@ public function prepare($builder)
         }
 
         if (empty($this->select) && !empty($this->join)) {
-            list(, $alias) = $this->getTableNameAndAlias();
+            [, $alias] = $this->getTableNameAndAlias();
```
Comment (Faryshta) [inline framework/db/ActiveQuery.php]: change to `$alias = $this->getTableNameAndAlias()[1];`?

File: framework/db/ActiveQuery.php [hunk 2]
```diff
@@ -438,7 +438,7 @@ private function buildJoinWith()
 
         $model = new $this->modelClass;
         foreach ($this->joinWith as $config) {
-            list ($with, $eagerLoading, $joinType) = $config;
+            [$with, $eagerLoading, $joinType] = $config;
```
Comment (Faryshta) [inline framework/db/ActiveQuery.php]: change line above to `foreach ($this->joinWith as [$with, $eagerLoading, $joinType] ) {`?

File: framework/db/ActiveQuery.php (hunk 1 above)
Comment (samdark) [inline framework/db/ActiveQuery.php]: Yes.

File: framework/db/ActiveQuery.php (hunk 2 above)
Comment (samdark) [inline framework/db/ActiveQuery.php]: Yes.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Please merge 2.1 into the pull request branch.

**Comment by Faryshta** [comment]
@samdark DONE!

The tests failing seem to be about missing exceptions and other functionalities, i don't think i can fix them on this PR.

**Comment by samdark** [comment]
Merged. Thanks!

**Comment by rob006** [comment]
php-cs-fixer has fixer for this, so such change could be easily automated. But merging this right now gives no value and will only create merge conflicts.

**Pull request merged**